themes/ansible-community/sass/_homepage-welcome-band.scss

```diff
@@ -0,0 +1,43 @@
+.homepage-welcome-band {
+  background-color: $white;
+}
+
+.welcome-row {
+  grid-column: span 2;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 40px;
+}
+
+.welcome-box-left {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 4rem;
+  p {
+    font-size: $font-md;
+  }
+}
+
+.welcome-box-right {
+  display: flex;
+  flex-direction: column;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  align-items: center;
+}
+
+@media screen and (max-width: 768px) {
+  .welcome-row {
+    grid-template-columns: 1fr;
+  }
+
+  .welcome-box-left {
+    padding-bottom: 2rem;
+  }
+
+  .welcome-box-right {
+    grid-column-start: initial;
+    grid-column-end: initial;
+  }
+}
```

Comment:
Please add an extra line at the end of the file.

Reply:
If you can implement that change for me I would appreciate it. My focus is on other activities today.

Reply:
Nevermind.  The change was implemented.